gatsby-config.js

```diff
@@ -80,6 +92,80 @@ module.exports = {
 		'gatsby-plugin-catch-links',
 		'gatsby-transformer-javascript-frontmatter',
 		'gatsby-plugin-react-helmet',
+		{
+			resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
+			options: {
+			  // Fields to index
+			  fields: [`title`, `description`, `memberDescriptions`, `members`],
+			  // How to resolve each field`s value for a supported node type
+			  resolvers: {
+				// For any node of type MarkdownRemark, list how to resolve the fields` values
+				MarkdownRemark: {
+					id: (node, getNode) => {
+						if (isNotEmpty(node.frontmatter.title)) {
+							const slug = node.fields.slug;
+							const newPath = slug.substring(1, slug.length-1);
+							return `${node.frontmatter.title}|${newPath}`;
+						} else {
+							const documentationJsComponentDescriptionNode = getNode(node.parent);
+							const documentationJsNode = getNode(documentationJsComponentDescriptionNode.parent);
+							const slug = documentationJsNode.fields.slug;
+							const newPath = slug.substring(1, slug.length-1);
+							const title = slug.substring(14, slug.length-1);
+							const name = documentationJsNode.name;
+							if (isNotEmpty(name) && name.includes('/')) {
+								compareslug = slug;
+								moduleName = name;
+								return `${title}|${newPath}`;
+							} else {
+								moduleName = "";
+							}
+						}
+					},
+					title: (node) => {
+						if (isNotEmpty(node.frontmatter.title)) {
+							return node.frontmatter.title;
+						} else {
+							if (isNotEmpty(moduleName)) {
+								return moduleName;
+							}
+						}
```

Comment:
```suggestion
						if (isNotEmpty(node.frontmatter.title) && isNotEmpty(moduleName)) {
							return node.frontmatter.title;
						} 
```
I think this if-else statement could be made simpler.